.changeset/pretty-dolls-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": minor
```

Comment:
Valid point @arboleya from that perspective we should make it breaking then, although we should discourage it's usage since it's intentionally not documented.

> Also, if this goes to master today, it's all good if it's a breaking change.

@Torres-ssf given `getTransferOperations` is not only [hidden](https://github.com/FuelLabs/fuels-ts/blob/st/fix/account-transfer-operation/packages/account/src/providers/transaction-summary/operations.ts#L397) but it's not used internally outside of `operations.ts` I think we should remove the _export_ for this as well  and include that it shouldn't be used in the release notes. Thoughts?

Reply:
At some point, we should review and trim the things we export for sure.

But I wouldn't bother with this now.

Reply:
Understood, I actually had started a discussion on this some time back so now just resurfacing [here](https://github.com/FuelLabs/fuels-ts/discussions/2429)